pyanaconda/modules/network/network.py

```diff
@@ -306,6 +313,35 @@ def is_connecting(self):
             log.debug("Connectivity state can't be determined, assuming not connecting.")
             return False
 
+    @property
+    def capabilities(self):
+        """Capabilities of the network backend."""
+        if not self.nm_available:
+            log.debug("Capabilities can't be determined.")
+            return []
```

Comment:
I think it does not hurt either and is kind of explicit. Yes we have such value ('[]') at 2 places but I was understanding it like in __init__ it can mean "initial unknown value" and here "value in case NM is not available" - which are the same in this implementation. But I don't mind removing it from here (or the __init__).

Reply:
I dont mind either, agreed about explicit.